src/toast/ops/stokes_weights/stokes_weights.py

```diff
@@ -24,33 +25,31 @@ class StokesWeights(Operator):
     aligned with the polarization sensitive direction.  An optional dictionary of
     pointing weight calibration factors may be specified for each observation.
 
-    For each observation, the cross-polar response for every detector is obtained from
-    the Focalplane, and if a HWP angle timestream exists, then a perfect HWP Mueller
-    matrix is included in the response.
+    If the hwp_angle field is specified, then an ideal HWP Mueller matrix is inserted
+    in the optics chain before the linear polarizer.  In this case, the fp_gamma key
+    name must be specified and each detector must have a value in the focalplane
+    table.
 
-    The timestream model is then (see Jones, et al, 2006):
+    The timestream model without a HWP in COSMO convention is:
 
     .. math::
-        d = cal \\left[\\frac{(1+eps)}{2} I + \\frac{(1-eps)}{2} \\left[Q \\cos{2a} + U \\sin{2a}\\right]\\right]
+        d = cal \\left[I + \\frac{1 - \\epsilon}{1 + \\epsilon} \\left[Q \\cos\\left(2\\alpha\\right) - U \\sin\\left(2\\alpha\\right) \\right] \\right]
 
-    Or, if a HWP is included in the response with time varying angle "w", then
-    the total response is:
+    When a HWP is present, we have:
 
     .. math::
-        d = cal \\left[\\frac{(1+eps)}{2} I + \\frac{(1-eps)}{2} \\left[Q \\cos{2a+4w} + U \\sin{2a+4w}\\right]\\right]
+        d = cal \\left[I + \\frac{1 - \\epsilon}{1 + \\epsilon} \\left[Q \\cos\\left(2(\\alpha - 2\\omega) \\right) + U \\sin\\left(2(\\alpha - 2\\omega) \\right) \\right] \\right]
 
-    The angle "a" in the above formalism is the angle (at each sample) between the
-    transformed X-axis of the detector frame and the local meridian of the coordinate
-    system.  This is computed by rotating the Z and X coordinate axes by the detector
-    pointing quaternion and then computing the rotation angle from meridian to this
-    polarization sensitive direction.  This means that "a" is positive in a
-    right-handed sense, since the rotated Z-axis points in the detector line of sight:
+    The detector orientation angle "alpha" in COSMO convention is measured in a
+    right-handed sense from the local meridian and the HWP angle "omega" is also
+    measured from the local meridian.  The omega value can be described in terms of
+    alpha, a fixed per-detector offset gamma, and the time varying HWP angle measured
+    from the focalplane coordinate frame X-axis:
 
     .. math::
-        v_m = vector parallel to local meridian
-        v_o = orientation vector (transformed x-axis)
-        v_d = direction vector (transformed z-axis)
-        a = atan2((v_m X v_o) \\dot v_d, v_m \\dot v_o)
+        \\omega = \\alpha + {\\gamma}_{H}(t) - {\\gamma}_{D}
```

Comment:
to be more explicit, how about : 
`\\omega = \\alpha + {\\gamma}_{HWP}(t) - {\\gamma}_{DET}` 
?

Reply:
Fixed.